PMM-2314 Instructing a user how to appropriately estimate disk space requireme… (#243)
percona/pmm: doc/source/deploy/server/ami.rst

   see its price. |pmm| comes for no cost, you may only need to pay for the
   infrastructure provided by |amazon|.

.. note::

>    Disk space used by |pmm-server| highly depends on the amount of hosts under

Reviewer: ```suggestion
   Disk space consumed by |pmm-server| depends on the number of hosts under
```

>    monitoring. There is an approximate formula tested on the PMM Demo <https://pmmdemo.percona.com/>`_

Reviewer: ```suggestion
   monitoring. Each environment will be unique, however consider modeling your data consumption based on PMM Demo <https://pmmdemo.percona.com/>`_
```

>    web site, which estimates allocated space as ~230MB/host/day, or ~6.9GB/host

Reviewer: ```suggestion
   web site, which consumes ~230MB/host/day, or ~6.9GB/host
```

>    in case of 30 day retention period, which is set by default. See `this blog post <https://www.percona.com/blog/2017/05/04/how-much-disk-space-should-i-allocate-for-percona-monitoring-and-management/>`_ for more

Reviewer: ```suggestion
at the default 30 day retention period. See `this blog post <https://www.percona.com/blog/2017/05/04/how-much-disk-space-should-i-allocate-for-percona-monitoring-and-management/>`_ for more
```

>    details about the |pmm-server| disk space consumption and its more precise

Reviewer: ```suggestion
   details.
```

>    planning.

Reviewer: ```suggestion

```
